src/App.css

```diff
@@ -1,16 +1,6 @@
-.App {
-  text-align: center;
-}
-
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
-}
-
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+html, body {
+  width: 100%;
```

Comment:
let's add normalize.css instead

Reply:
Sure, I linked normalized CSS to the App.js file and deleted this custom thing from App.css